citrination_client/data/tests/test_data_client.py

```diff
@@ -21,8 +21,8 @@ def random_string():
 # one dataset with the same name
 dataset_name = "Tutorial dataset " + random_string()
 dataset_id = client.create_dataset(name=dataset_name, description="Dataset for tutorial").id
-test_file_root = './citrination_client/data/tests/test_files/'
-test_file_data_root = './citrination_client/data/tests/test_files/data/'
+test_file_root = 'citrination_client/data/tests/test_files/'
```

Comment:
Why'd you remove these `./`?  Could you leave one in and one out for coverage?

Reply:
Yep.